.envrc

```diff
@@ -218,23 +193,29 @@ fi
 make node-version-check
 
 if [ ! -x "node_modules/.bin/webpack" ]; then
-    info "You don't seem to have yarn packages installed."
-    advice_install_yarn_pkgs
+    warn "You don't seem to have yarn packages installed."
+    commands_to_run+=("make install-js-dev")
 fi
 
 PATH_add node_modules/.bin
 
-### Overrides ###
+# These are commands that can take a significant ammount of time
+if [ ${#commands_to_run[@]} -ne 0 ]; then
```

Comment:
This is the block in charge of listing all commands that are needed in order to get the environment up-to-date.